pycargoebuild/cargo.py

```diff
@@ -299,6 +300,7 @@ def get_package_metadata(f: typing.BinaryIO,
         name=pkg_meta["name"],
         version=pkg_version,
         license=pkg_license,
+        features=cargo_toml.get("features", None),
```

Comment:
```suggestion
        features=cargo_toml.get("features"),
```

`None` is the default.